src/main/webapp/resources/js/styles/borders.js

```diff
@@ -0,0 +1,10 @@
+import { grey4 } from "./colors";
+/**
+ * @fileOverview Borders used in .js and .jsx files need to be standardized
+ * using the values contained in this file.
+ *
+ * PLEASE TO NOT ADD TO OR CHANGE VALUES IN THIS FILE WITHOUT SPEAKING TO THE
+ * LEAD ON INTERFACE DESIGN.
+ */
+
+export const border1 = `solid 1px ${grey4}`;
```

Comment:
I am not really sure about just calling it `border1`.  It make sense with the colours because the get more intense as the number increases.  This does not describe what it is, come up with something better ;)

Reply:
Updated in 17c23fd74ab4bdcfe6c55f80cbe3c82226c59dab